bids-validator/validators/tsv/tsvParser.js

```diff
@@ -3,7 +3,11 @@
  * Module for parsing TSV (and eventually other formats)
  */
 
-const normalizeEOL = str => str.replace('\r\n', '\n').replace('\r', '\n')
+const normalizeEOL = str =>
+  str
+    .replace('\r\n', '\n')
+    .replace('\r', '\n')
+    .replace(/\uFEFF/g, '')
```

Comment:
The existing function is self-documenting, and this change makes it less clear. What about a second function?

```suggestion
const stripBOM = str => str.replace(/^\uFEFF/, '')
const normalizeEOL = str => str.replace('\r\n', '\n').replace('\r', '\n')
```

Also, I use `/^\uFEFF/` since BOM should only be valid at the start of a stream.

Reply:
Truth and wisdom, all of it.